### DISCUSSION TITLE
请问如何提升windows自启动的启动顺序或者启动速度？

### DISCUSSION THREAD

**Discussion by mcitem** [clash-verge-rev/clash-verge-rev#1541] [Q&A]
开着自启动基本都要好几秒才完成启动，又因为有系统代理，导致短时间内会断网

**Comment by ghost** [comment]
是有这个问题，启动比较慢。另外平时打开也挺慢的。。

**Comment by iwalke** [comment]
+1，启动太慢了，希望优化一下。几本所有开机自启的软件都启动完成了，clash-verge-rev 还没启动，太慢了。
